cg/cli/clean.py

```diff
@@ -127,6 +188,103 @@ def scout_finished_cases(
         context.invoke(hk_alignment_files, bundle=bundle, yes=yes, dry_run=dry_run)
 
 
+@clean.command("hk-case-bundle-files")
+@click.option(
+    "--days-old",
+    type=int,
+    default=365,
+    help="Clean all files with analysis dates older then given number of days",
+)
+@click.option("-d", "--dry-run", is_flag=True, help="Shows cases and files that would be cleaned")
+@click.pass_context
+def hk_case_bundle_files(context: CGConfig, days_old: int, dry_run: bool = False):
+    """Clean up all files for all pipelines but those whitelisted"""
+    status_db: Store = context.obj.status_db
+    housekeeper_api: HousekeeperAPI = context.obj.housekeeper_api
+    before: datetime = datetime.now() - timedelta(days=days_old)
```

Comment:
Should we turn this into a function that can be reused. Looks like you are testing the logic in a function (get_date_days_ago) only defined in the test? But not using an actual function here? I might have misunderstood something....

Reply:
replaced with calls to the function